### PR TITLE
[move-docs] Revise step 7 and 8

### PR DESCRIPTION
**Pull request by rahxephon89**
## Motivation

Revision of step 7 and 8

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: language/documentation/hackathon-tutorial/README.md [hunk 1]
```diff
@@ -493,30 +493,64 @@ to reuse code since the generic library module already provides a standard imple
...
+    }
+```
 
-#### Method `withdraw`
+Informally speaking, the block `spec balance_of {...}` contains the property specification of the method `balance_of`. Let's first run the prover using the command `move package prove` inside the `sources` directory, which outputs the following error information:
 
-The signature of the method `withdraw` is given below:
 ```
-fun withdraw<CoinType>(addr: address, amount: u64) : Coin<CoinType> acquires Balance
+error: abort not covered by any of the `aborts_if` clauses
```
Comment (emmazzz) [inline language/documentation/hackathon-tutorial/README.md:509]: The code in `step_7` already contains the aborts_if spec so when users run `move package prove` they won't actually see the error. Maybe it's better to drop this aborts_if from `step_7/BasicCoin/sources/BasicCoin.move` and have them add it themselves?

**Review by emmazzz — APPROVED** [review]
Left a comment. Otherwise looks good to me :)

**Review by tzakian — APPROVED** [review]
LGTM!

**Pull request merged**